Purchases/RCPurchases+Protected.h

```diff
@@ -19,7 +19,8 @@ NS_ASSUME_NONNULL_BEGIN
                                     backend:(RCBackend *)backend
                             storeKitWrapper:(RCStoreKitWrapper *)storeKitWrapper
                          notificationCenter:(NSNotificationCenter *)notificationCenter
-                               userDefaults:(NSUserDefaults *)userDefaults;
+                               userDefaults:(NSUserDefaults *)userDefaults
+                               observerMode:(Boolean)observerMode;
```

Comment:
use BOOL not Boolean.

https://nshipster.com/bool/

Reply:
oops